src/autonomous_company/tools/marketing_tools.py

```
@@ -0,0 +1,216 @@
+
+    def _run(self, topic: str, audience: str, timeframe_weeks: int = 4) -> str:
+        """Generate a content calendar."""
+        formats = ["blog_post", "social_media", "email_newsletter", "video", "infographic"]
+        channels = ["website", "linkedin", "twitter", "email", "youtube"]
+
+        start = date.today()
+        pieces = []
+
+        for week in range(timeframe_weeks):
+            week_start = start + timedelta(weeks=week)
+            for i, (fmt, channel) in enumerate(zip(formats[:3], channels[:3])):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)